src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
```

Comment:
- [ ] Kindly add back the App.js component,  For this milestone please:

- The App.js component should be class-based and The App.js(main component) should have the imported Calculator.js component   
- Currently, you have imported your calculator component in your index.js please remove it, your App.js should be imported in the index.js